registries/adapters/openshift_adapter_test.go

```diff
@@ -89,9 +91,17 @@ const AuthResponse = `
   "issued_at": "2018-03-27T19:54:19Z"
 }`
 
-func TestOpenShiftName(t *testing.T) {
-	ocpa := OpenShiftAdapter{}
-	ft.Equal(t, ocpa.RegistryName(), "openshift", "openshift name does not match `openshift`")
+func TestRegistryName(t *testing.T) {
+	url, err := url.Parse(TestURL)
+	if err != nil {
+		t.Fatal("Error: ", err)
+	}
```

Comment:
Consider using a table for the test cases, this will allow us to easily add new urls if we want to test other conditions:
```golang
func TestRegistryName(t *testing.T) {

    testcases := []struct {
        urlstr string
        result string
    }{
        {"http://foo.redhat.com", "foo.redhat.com"},
        {"https://registry.reg-aws.openshift.com:443", "registry.reg-aws.openshift.com:443"},
    }

    for _, testcase := range testcases {
        url, err := url.Parse(testcase)
        if err != nil {
            t.Fatal("Error: ", err)
        }

        ocpa := OpenShiftAdapter{
            Config: Configuration{
                URL: url,
            },
        }
        ft.Equal(t, ocpa.RegistryName(), result, "the registry name returned did not match expected value")
    }
}
```